cmake/Modules/ConfigureGoogleTest.cmake

```diff
@@ -1,10 +1,17 @@
 set(GTEST_ROOT "${CMAKE_BINARY_DIR}/googletest")
 
 set(GTEST_CMAKE_ARGS " -Dgtest_build_samples=ON" 
-                     " -DCMAKE_VERBOSE_MAKEFILE=ON"
-                     " -DCMAKE_C_FLAGS=-D_GLIBCXX_USE_CXX11_ABI=0"      # enable old ABI for C/C++
-                     " -DCMAKE_CXX_FLAGS=-D_GLIBCXX_USE_CXX11_ABI=0")   # enable old ABI for C/C++
-
+                     " -DCMAKE_VERBOSE_MAKEFILE=ON")
```

Comment:
Do we always want this on or is this a remnant of debugging?

Reply:
Removed this and the samples option.